Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogScreen.h

```diff
@@ -12,7 +12,7 @@
 #include <ScreenWidget.h>
 #include <AzCore/std/smart_ptr/unique_ptr.h>
 #include <AzToolsFramework/UI/Notifications/ToastNotificationsView.h>
-
+#include "GemInfo.h"
```

Comment:
though quotes are more in line with most coding standards and [what quote vs angle bracket denotes](https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html), [our own coding guidelines use angle brackets](https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#including-headers) and that will be more consistent with the rest of the code in this project.
```suggestion
#include <GemInfo.h>
```